Sources/dep/Git.swift

```diff
@@ -91,10 +91,14 @@ class Git {
         }
 
         do {
+            var environment = [String:String]()
+            if let sshAuthSockPath = getenv("SSH_AUTH_SOCK") { //for auto ssh authentication if possible
+                environment["SSH_AUTH_SOCK"] = sshAuthSockPath
```

Comment:
There are also other env variables that might be nice to replicate.

- `GIT_ASKPASS` ([git-credentials](https://git-scm.com/docs/gitcredentials))
- `SSH_ASKPASS`
- `XDG_CONFIG_HOME` (used for `git-credenitials` lookup)

In which, I wonder why not just inherit all environ variables?